pkgs/development/python-modules/fusepy/default.nix

```diff
@@ -18,7 +19,9 @@ buildPythonPackage rec {
   # No tests included
   doCheck = false;
 
-  patchPhase = ''
+  # On macOS, users are expected to install macFUSE. This means fusepy should
+  # be able to find libfuse in /usr/local/lib.
+  patchPhase = lib.optionalString (!stdenv.isDarwin) ''
```

Comment:
```suggestion
  patchPhase = lib.optionalString !stdenv.isDarwin ''
```

Reply:
I get a syntax error without the parens:

```
error: syntax error, unexpected '!', expecting ';', at /Users/midchildan/Documents/src/repos/github.com/NixOS/nixpkgs/pkgs/development/python-modules/fusepy/default.nix:24:35
```

Reply:
Yes. It is required.